frontend/src/component/styles/color.scss

```diff
@@ -1,6 +1,7 @@
 $color-primary: #e91e63;
 $color-white: #fff;
 $color-dark-gray: #343434;
+$color-light-blue: lightblue;
```

Comment:
```suggestion
$color-highlight: lightblue;
```

Reply:
If we name it light-blue, when we change the value, we would have to change the name of this variable across the codebase.